deployment/disable_maintenance_mode.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+rm /var/www/html/maintenance.css
```

Comment:
_If_ we decide to add `set -e`, these should maybe be `rm -f` so that the script still works after a change of this list / if this script is used for a first setup. But that depends on how you want this to work in your workflow and how it is used :)